proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,226 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **About**
+  - Kubeflow
+  - Use Cases
+  - Contributing to Kubeflow
+  - Community
+  - Google Summer of Code
+  - Docs
+  - Style Guide for Kubeflow docs
```

Comment:
Do we need to include [events calendar](https://www.kubeflow.org/docs/about/events/) ?

Reply:
Nice catch!